### PR TITLE
🏗 Fold visual diff test errors and warning on Travis

### PR DESCRIPTION
**Pull request by danielrozenberg**
Followup fix for comment https://github.com/ampproject/amphtml/pull/21852#pullrequestreview-227563859

### CODE REVIEW

File: build-system/tasks/visual-diff/index.js [hunk 1]
```diff
@@ -521,10 +521,13 @@ async function snapshotWebpages(percy, browser, webpages) {
     await sleep(WAIT_FOR_TABS_MS);
   }
   log('travis', '\n');
-  if (isTravisBuild()) {
+  if (isTravisBuild() && testErrors.length > 0) {
     testErrors.sort((a, b) => a.name.localeCompare(b.name));
-    // TODO(danielrozenberg): add Travis log folding.
+    log('info', colors.yellow('Tests warnings and errors:'),
+        'expand this section');
+    exec('echo travis_fold:start:lint_results && echo');
```
Comment (rsimha) [inline build-system/tasks/visual-diff/index.js]: The string after `travis_fold:start:` must match the string after `travis_fold:end:`

**Comment by danielrozenberg** [comment]
That explains why this isn't working for me >__<

File: build-system/tasks/visual-diff/index.js (hunk 1 above)
Comment (danielrozenberg) [reply build-system/tasks/visual-diff/index.js]: That explains why this isn't working for me >__<

**Comment by danielrozenberg** [comment]
Sample folded log: https://travis-ci.org/ampproject/amphtml/jobs/523592806#L1550

**Pull request merged**